File: benchmark/results/java-invocation-inline/run-001/b-baseline/output/com/metaengine/demo/shipping/services/ShipmentService.java
package com.metaengine.demo.shipping.services;

import com.metaengine.demo.shipping.aggregates.Shipment;
import java.util.List;

/** Application service exposing shipment operations. */
public class ShipmentService {
    /** Create a shipment from the provided partial input. */
    public Shipment create(Shipment input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a shipment by id. Returns null when not found. */
    public Shipment findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List up to {@code limit} shipments. */
    public List<Shipment> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete a shipment by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/ts-script-sonnet/run-005/b-baseline/output/src/domain/shipping/services/ShipmentService.ts
import { Shipment } from '../aggregates/Shipment';

/** Application service for managing shipment operations. */
export class ShipmentService {
  create(input: Partial<Shipment>): Shipment {
    void input;
    throw new Error('not implemented');
  }

  findById(id: string): Shipment | null {
    void id;
    throw new Error('not implemented');
  }

  list(limit: number): Shipment[] {
    void limit;
    throw new Error('not implemented');
  }

  delete(id: string): void {
    void id;
    throw new Error('not implemented');
  }
}
